tests/fixtures/test_vectors.json

```diff
@@ -0,0 +1,66 @@
+{
```

Comment:
What was used to generate these fixtures ?

Reply:
https://thoralf-m.github.io/iota-ed25519-address-wasm/ :trollface:

Reply:
And what does this use under the hood ? Ideally fixtures need to be generated from a completely different implementation.

Reply:
crypto.rs and bee-message
It's also compared with hornet from https://github.com/gohornet/hornet/blob/develop/private_tangle/private_tangle_keys.md and https://github.com/iotaledger/tips/blob/add-smr-hrps/tips/TIP-0031/tip-0031.md